data/Moyo-Made.json

```
@@ -0,0 +1,19 @@
+{
+  "name": "Moyo-Made",
+  "bio": "Software Developer | JavaScript | React | Open to opportunities"
```

```suggestion
  "bio": "Software Developer | JavaScript | React | Open to opportunities",
```
please add a comma here